WcaOnRails/app/controllers/static_pages_controller.rb

```diff
@@ -6,8 +6,12 @@ class StaticPagesController < ApplicationController
   def home
   end
 
-  def delegates
+  def delegates_data
     @delegates = User.where.not(delegate_status: nil)
```

Comment:
It is not necessary to prefix variables with `@` if there is no view template that references these (global) variables. Using `delegates = ...` should suffice

Reply:
Done.